components/core/src/ffi/encoding_methods.tpp

```diff
@@ -464,6 +464,60 @@ namespace ffi {
 
         return false;
     }
+
+    template <typename encoded_variable_t>
+    bool wildcard_match_encoded_vars (
+            std::string_view logtype,
+            encoded_variable_t* encoded_vars,
+            int encoded_vars_length,
+            std::string_view wildcard_var_placeholders,
+            const std::vector<std::string_view>& wildcard_var_queries
+    ) {
+        auto wildcard_var_queries_len = wildcard_var_queries.size();
```

Comment:
would be nice to check that wildcard_var_placeholders.length() = wildcard_var_queries.size()?

Reply:
Good idea! Added some parameter validation.